CHANGELOG.md

```diff
@@ -28,6 +28,7 @@
 ## Fixes
 
 - Fixed an issue where some types on the `Call` and `Voice` classes were being incorrectly exported. Types and references to `addEventListener` are instead now correctly exported as `addListener`.
+- Fixed an issue where available audio devices were incorrectly emitted and returned by the SDK.
```

Comment:
Were the audio devices incorrectly returned 100% of the time, or only on certain scenario? I think we should include that information.

Reply:
Only sometimes, good point. Should have only happened on Android when reloading bundles, probably only in development environments but maybe in prod environments.